Gems/Stars/Code/Source/StarsComponentController.cpp

```diff
@@ -68,7 +68,6 @@ namespace AZ::Render
 
     void StarsComponentController::RegisterFeatureProcessor(EntityId entityId)
```

Comment:
Nit, maybe update the function name to EnableFeatureProcessor?

Reply:
good point, will do.